### ISSUE TITLE
Dotnet tool install cannot handle space+unicode paths

### ISSUE THREAD

**Issue by MikaelUmaN** [dotnet/sdk#2455]
Please see original issue:

https://github.com/fsharp/FAKE/issues/2054#issuecomment-410527662

**To reproduce**

```
matth@DESKTOP-FQBAN56 E:\Projects\FAKE\temp\Mikael Öhman\Reproduce
$ dotnet tool install fake-cli --tool-path ./.fake --version 5.*
MSBUILD : Error MSB1006: Property is not valid.
Switch: hman\Reproduce\.fake\.store\.stage\24kqfmg2.aq2
For switch syntax, type "MSBuild /help"
The tool package could not be restored.
Tool 'fake-cli' failed to install. This failure may have been caused by:

* You are attempting to install a preview release and did not use the --version option to specify the version.
* A package by this name was found, but it was not a .NET Core tool.
* The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
* You mistyped the name of the tool
```

**Expected result**
Fake CLI tools are installed in the folder with path involving spaces and unicode characters (e.g. Ö).

**Actual result**
The error seen above.

**Comment by livarcocc** [comment]
@MikaelUmaN which version of the SDK were you using?

@wli3 is this the bug we have fixed already?

**Comment by wli3** [comment]
It should be fixed in 2.1302 https://github.com/dotnet/cli/commit/5218c4bcf52441075ca4d5b8fa7ed860c99ea454

**Comment by matthid** [comment]
@wli3 I guess you mean 2.1.302. Doesn't look like it is. Just to be sure I uninstalled all other SDKs via programs and features:

```
E:\Projects\FAKE\temp\Mikael Öhman\Reproduce>dotnet tool install fake-cli --tool-path .fake
MSBUILD : error MSB1006: Property is not valid.
Switch: hman\Reproduce\.fake\.store\.stage\5oxp3oza.ehb
For switch syntax, type "MSBuild /help"
The tool package could not be restored.
Tool 'fake-cli' failed to install. This failure may have been caused by:

* You are attempting to install a preview release and did not use the --version option to specify the version.
* A package by this name was found, but it was not a .NET Core tool.
* The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
* You mistyped the name of the tool.

E:\Projects\FAKE\temp\Mikael Öhman\Reproduce>dotnet --info
.NET Core SDK (reflecting any global.json):
 Version:   2.1.302
 Commit:    9048955601

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.17134
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.302\

Host (useful for support):
  Version: 2.1.2
  Commit:  811c3ce6c0

.NET Core SDKs installed:
  2.1.302 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.2 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.2 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.2 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```

**Comment by wli3** [comment]
I am so sorry, though the commit count is higher, dotnet sdk 2.1.302 release does not have this change. The code change is in latest 2.1.3xx branch and all the other branches. And I cannot repo it in current 2.1.4xx "merge build". It should be available for the next 2.1.3xx build.
